server/src/main/resources/db/common/V115__alter_volunteering_relationship.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE volunteering_relationship
```

Comment:
Need to rename this file to increment to 116 before merging. 115 has now been taken in develop

Reply:
Done.